dom/fuzzer/modules/web-speech.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * WebSpeech References
+ *
+ * Implementation: https://bugzilla.mozilla.org/show_bug.cgi?id=1003439
+ * WebIDL:
+    dom/webidl/SpeechGrammarList.webidl
+    dom/webidl/SpeechGrammar.webidl
+    dom/webidl/SpeechRecognitionAlternative.webidl
+    dom/webidl/SpeechRecognitionError.webidl
+    dom/webidl/SpeechRecognitionEvent.webidl
+    dom/webidl/SpeechRecognitionResultList.webidl
+    dom/webidl/SpeechRecognitionResult.webidl
+    dom/webidl/SpeechRecognition.webidl
+    dom/webidl/SpeechSynthesisErrorEvent.webidl
+    dom/webidl/SpeechSynthesisEvent.webidl
+    dom/webidl/SpeechSynthesisUtterance.webidl
+    dom/webidl/SpeechSynthesisVoice.webidl
+    dom/webidl/SpeechSynthesis.webidl
+ * Mochitests: dom/media/webspeech/recognition/test
+ *             dom/media/webspeech/synth/test
+ * animation/test/css-animations
+ * Specification: https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html
+ * MDN: https://developer.mozilla.org/en-US/docs/Web/API/Web_Speech_API
+ *
+**/
+var fuzzerWebSpeech = (function() {
+  var utterances = [];
+  var recognitions = [];
+  var grammars = [];
+  var grammarLists = [];
+  var thingsToSay = ['Stop, Dave.', 'Would you like to play a game?', 'Destruct sequence engaged.', fuzzValues.texts];
+  var thingsToGrammar = ["#JSGF V1.0; grammar test; public <simple> = hello ;"]; // XXX fuzz this...
+
+  /*
+  ** Main
+  */
+  var makeCommand = function(){
+    var cmd = [];
+
+    fuzzInternalErrorsAreBugs = false;
+    Random.choose([
```

Comment:
Random.choose for side effects through Random.pick? Not sure whether "yikes" or "awesome".